### ISSUE TITLE
Can't reproduce the performance on the ScanNet dataest.

### ISSUE THREAD

**Issue by lqzhao** [udaykusupati/Normal-Assisted-Stereo#16]
Hi, I tried to train the model on the ScanNet dataset, but we got a very worse result, compared with the result reported in the paper. We got 0.2440 on the **abs_rel** term on the validation set. However, we can get 0.1179 when using the pretrained model provided by the auther. What makes the huge performance gap? Is the pretrained model only trained on the ScanNet?

Our training settings: 4 NVIDIA GTX 2080Ti GPUs with a batch size of 12, pytorch 1.3.1, CUDA 10.2
Thanks
